src/main/java/de/tum/in/www1/artemis/service/TextSubmissionService.java

```diff
@@ -44,24 +50,58 @@ public TextSubmission save(TextSubmission textSubmission, TextExercise textExerc
         // update submission properties
         textSubmission.setSubmissionDate(ZonedDateTime.now());
         textSubmission.setType(SubmissionType.MANUAL);
-        textSubmission.setParticipation(participation);
+
+        boolean isExampleSubmission = textSubmission.isExampleSubmission() != null && textSubmission.isExampleSubmission();
```

Comment:
I do not understand the && operator here. You are testing the same boolean value twice

Reply:
I have to be sure it is not null, I replaced with `textSubmission.isExampleSubmission() == Boolean.TRUE`